arbitrum-docs/node-running/gentle-introduction-run-node.md

```diff
@@ -1,19 +1,18 @@
 ---
-title: "A gentle introduction to Arbitrum nodes"
+title: "A gentle Introduction to Arbitrum Nodes"
 sidebar_label: "A gentle introduction: Arbitrum nodes"
 description: "Arbitrum is a suite of Ethereum L2 scaling solutions supported by a decentralized network of nodes. This guide introduces you to Arbitrum's node types and how they work together to scale Ethereum."
-author: dzgoldman
+author: mahsamoosavi
 ---
 
-import UnderConstructionPartial from '../partials/_under-construction-banner-partial.md'; 
+In order to be able to _ineract with_ or _build applications on_ any of the Arbitrum chains, you need access to the corresponding Arbitrum node. Options are:
 
-<UnderConstructionPartial />
+- You can use third party node providers (see the list [here](./node-providers.mdx)) to get RPC access to fully-managed nodes 
+- You can run your own Arbitrum node, especially if you want to always know the state of the Arbitrum chain. 
 
+Here, you can find resources that help you run different types of Arbitrum nodes:
 
-<!-- todo: build bridge-specific gentle intro, and reuse content from the OG gentle intro whenever possible 
-
-import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
-
-<GentleIntroPartial />
-
--->
\ No newline at end of file
+- Step-by-step instructions for running different Arbitrum nodes
```

Comment:
I would add links here. For instructions there are several pages, not sure how to handle it, but maybe you can list them after a colon, and place individual links for the most common type of nodes like ": full Nitro node, full Classic node, local dev node, feed relay, and others."
Also, with the new changes there aren't concepts in this section. We can remove that line now and add it back later if we create a concept page.